packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -55,6 +55,14 @@ const AssetSummary = observer(() => {
         );
     }
 
+    // measure performance metrics
```

Comment:
@sergei-deriv How could you make sure this component is fully loaded?

Reply:
@mahdiyeh-deriv this line will be called only when all conditions for loader is false and we should show total asset value